openapi/components/schemas/RiskScore/RiskScoreBlocklist.yaml

```
@@ -0,0 +1,43 @@
+type: object
+description: Risk score blocklist configuration.
+properties:
+  address:
+    description: House number and ZIP code of the customer's address.
+    allOf:
+      - $ref: ./RiskScoreBlocklistType.yaml
+  bank-account:
+    description: Bank account being charged.
+    allOf:
+      - $ref: ./RiskScoreBlocklistType.yaml
+  bin:
+    description: Bank Identification Number (BIN) of the payment card being charged.
+    allOf:
+      - $ref: ./RiskScoreBlocklistType.yaml
+  country:
+    description: Country where the customer is making the purchase, and of the payment card billing address.
+    allOf:
+      - $ref: ./RiskScoreBlocklistType.yaml
+  customer-id:
+    description: Rebilly customerId of the customer making the purchase.
+    allOf:
+      - $ref: ./RiskScoreBlocklistType.yaml
+  email:
+    description: Email address of the customer making the purchase.
+    allOf:
+      - $ref: ./RiskScoreBlocklistType.yaml
+  email-domain:
+    description: Email address domain of the customer making the purchase.
+    allOf:
+      - $ref: ./RiskScoreBlocklistType.yaml
+  fingerprint:
+    description: String holding information about the software and the device used by the customer making the purchase.
```

suggestion: suggestion to provide more detail
```suggestion
    description: |-
      Fingerprint of the user's device.
      A device fingerprint is a unique token that is used to identify the user.
      The device fingerprint is generated based on device attributes,
      such as: hardware, software, IP address, language, browser, and more.
```